doc/classes/Node.xml

```diff
@@ -761,6 +761,7 @@
 			<param index="1" name="recursive" type="bool" default="true" />
 			<description>
 				Sets the node's multiplayer authority to the peer with the given peer ID. The multiplayer authority is the peer that has authority over the node on the network. Useful in conjunction with [method rpc_config] and the [MultiplayerAPI]. Inherited from the parent node by default, which ultimately defaults to peer ID 1 (the server). If [param recursive], the given peer is recursively set as the authority for all children of this node.
+				[b]Important:[/b] This does [b]not[/b] automatically replicate the new authority to other peers! It is your responsibility to do so, e.g. though [member MultiplayerSpawner.spawn_function], an RPC, or even using a [MultiplayerSynchronizer].
```

Comment:
Maybe use `Warning` instead of `Important`, also do we usually use exclamation points in the doc? I think it might be a bit excessive here